sensors/migrations/0002_auto_20190404_1424.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.5 on 2019-04-04 14:24
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sensors', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='sensormeasurement',
+            name='type',
+            field=models.CharField(choices=[('temperature', 'Temperature'), ('humidity', 'Humidity'), ('sound', 'Sound'), ('people', 'People count')], max_length=16),
```

Comment:
This could be merged with migration 0001

Reply:
It _could_, but optimizing migrations at this stage is somewhat premature. Keeping the git history logical is as important at this point, IMO.